src/main.rs

```diff
@@ -117,19 +117,7 @@ impl Snare {
     ///
     /// If `msg` contains a `NUL` byte.
     fn fatal(&self, msg: &str) -> ! {
-        if self.daemonised {
```

Comment:
So if I understand correctly, this removed code worked, but was unnecessary, as we can just delegate the logic to `fatal()`?

Reply:
Previously `fatal` wasn't powerful enough, but because it now takes a `daemonised` argument, it can subsume this call.